FriendlyMenu/DataAccessors/RestaurantHourDataAccessor.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Entities;
+using Interfaces.DataAccessors;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using DataModels;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using DbModels;
+
+namespace DataAccessors
+{
+    class RestaurantHourDataAccessor : IRestaurantHourDataAccessor
```

Comment:
Why do we need this?

Reply:
all the data accessors other than RestaurantDataAccessor arent not include in the project. thats the old code. I can Delete it now.